test/ctia/entity/feed_test.clj

```diff
@@ -188,12 +193,14 @@
                              (:judgements valid-response-body)))))
 
             ;;teardown
-            (is (= 200
-                   (:status
-                    (helpers/PUT app
-                                 (str "ctia/feed/" (:short-id feed-id))
-                                 :body feed
-                                 :headers {"Authorization" "45c1f5e3f05d0"}))))))))))
+            (is (= 200 (:status
+                        (helpers/PUT app
+                                     (str "ctia/feed/" (:short-id feed-id))
+                                     :body feed
+                                     :headers {"Authorization" "45c1f5e3f05d0"}))))))))))
+(use-fixtures :once
+  (join-fixtures [validate-schemas
+                  whoami-helpers/fixture-server]))
```

Comment:
Why did you move the `use-fixtures` block?

Reply:
oops! fixed that!